### PR TITLE
add google noto + google noto emoji and remove image emojis; fixes #483

### LINKED ISSUE #483: Emojis in input field are displayed as tofu

### PR DESCRIPTION
**Pull request by Simon-Laux**
closes #483 
It also adds Google Noto Mono as fallback to make tofu chars also in other languages much more unlikely.

### CODE REVIEW

**Comment by Simon-Laux** [comment]
We also need someone that can test it on a mac just to be sure that it works there too

File: src/renderer/components/conversations/Emojify.js [hunk 1]
```diff
@@ -1,33 +1,19 @@
 const React = require('react')
 const classNames = require('classnames')
-const is = require('@sindresorhus/is')
```
Comment (ralphtheninja) [inline src/renderer/components/conversations/Emojify.js]: We still use this elsewhere? Or can it be removed now?

Comment (ralphtheninja) [reply src/renderer/components/conversations/Emojify.js]: screenshot from 2019-01-05 20-46-28

Doesn't seem like it.

Comment (Simon-Laux) [reply src/renderer/components/conversations/Emojify.js]: I'll remove it

**Pull request merged**